frontend/app/.ic-assets.json5

```diff
@@ -8,7 +8,7 @@
             "Strict-Transport-Security": "max-age=31536000; includeSubDomains",
             "X-Content-Type-Options": "nosniff",
             "X-XSS-Protection": "1; mode=block",
-            "Content-Security-Policy": "frame-ancestors https://4hy5z-qaaaa-aaaal-addrq-cai.icp0.io https://desktop.windoge98.com https://vault-bet.com https://www.vault-bet.com https://gooble.app  https://signalsicp.com https://kjfeq-waaaa-aaaan-qedva-cai.icp0.io https://spyzr-gqaaa-aaaan-qd66q-cai.icp0.io https://221bravo.app https://ht7v7-iaaaa-aaaak-qakga-cai.icp0.io https://mdocx-gyaaa-aaaak-qcbsq-cai.icp0.io https://calm-pasca-49d7be.netlify.app http://localhost:5173 https://zexzi-jyaaa-aaaam-abj3q-cai.icp0.io https://xw4dq-4yaaa-aaaam-abeuq-cai.ic0.app https://zkpoker.app https://panoramablock.com https://zdgud-kqaaa-aaaal-ajn4q-cai.icp0.io https://okowr-oqaaa-aaaag-qkedq-cai.icp0.io https://pre.konecta.one https://konecta.one https://y7mum-taaaa-aaaag-qklxq-cai.icp0.io;",
+            "Content-Security-Policy": "frame-ancestors https://4hy5z-qaaaa-aaaal-addrq-cai.icp0.io https://desktop.windoge98.com https://vault-bet.com https://www.vault-bet.com https://gooble.app  https://signalsicp.com https://kjfeq-waaaa-aaaan-qedva-cai.icp0.io https://spyzr-gqaaa-aaaan-qd66q-cai.icp0.io https://221bravo.app https://ht7v7-iaaaa-aaaak-qakga-cai.icp0.io https://mdocx-gyaaa-aaaak-qcbsq-cai.icp0.io https://calm-pasca-49d7be.netlify.app http://localhost:5173 https://zexzi-jyaaa-aaaam-abj3q-cai.icp0.io https://xw4dq-4yaaa-aaaam-abeuq-cai.ic0.app https://zkpoker.app https://panoramablock.com https://zdgud-kqaaa-aaaal-ajn4q-cai.icp0.io https://okowr-oqaaa-aaaag-qkedq-cai.icp0.io https://pre.konecta.one https://konecta.one https://y7mum-taaaa-aaaag-qklxq-cai.icp0.io https://e4tvt-6yaaa-aaaao-a3sdq-cai.icp0.io/lair;",
```

Comment:
Could you remove the path element /lair in both places? It should just be the origin.

Reply:
Yes I will do this now. I’m sorry, im a neophyte. I thought something looked wrong!

Reply:
Ok I somehow made two new prs with the corrections, so I guess you can close this one? Lol idk what I’m doing.

Reply:
ha - no worries - I will sort it out